client/components/keyed-suggestions/index.jsx

```diff
@@ -228,8 +229,18 @@ class KeyedSuggestions extends React.Component {
 				// concatenate mathing and non matchin - this is full set of filters just reordered.
 				filtered[ key ] = [ ...matchingSorted, ...parts[ 1 ] ];
 			} else {
+				// Try a full match first and try substring matches
+				let multiRegex = filterTerm;
```

Comment:
We've got the Fuse library for just this sort of searching, but I had a play and it's more effort that I expected.

It's not expecting the key:value pairs the way we'd expect, 

```node
Fuse = require('fuse.js');
// terms = copy and paste JSON.stringify(terms)
Object.values = Object.values || ((obj) => Object.keys(obj).map((k) => obj[k])) // node doesn't have Object.values
fuse = new Fuse(Object.values(terms.subject), options);
results = fuse.search('Boda')
[ { item: 
     { name: 'Bodas',
       description: 'Con este diseño romántico y organizado, puedes compartir datos importantes, configurar una cuenta atrás para tu gran día y recopilar confirmaciones de asistencia mediante un formulario de contacto.' },
    score: 0.001 },
  { item: 
     { name: 'Anuncio',
       description: 'Temas para bodas, reuniones, lanzamientos de productos: todo lo que quieras anunciar a tus familiares, amigos o clientes. Comparte fotos, fechas, preguntas frecuentes, instrucciones, indicaciones y mucho más.' },
    score: 0.11 },
  { item: 
     { name: 'Moda',
       description: 'Hauls de ropa, entradas del #OOTD, reseñas de desfiles: elige entre una gama de temas elegantes pensados para mostrar fotos y destacar tu feed de Instagram. Algunos son minimalistas y otros tienen toques adicionales de diseño gráfico.' },
    score: 0.1475 },
  { item: 
     { name: 'Colaboración',
       description: 'Temas diseñados para ayudar a los grupos a comunicarse y colaborar con clientes, estudiantes, amigos o compañeros de trabajo, y para planear proyectos, discutir temas urgentes o simplemente charlar.' },
    score: 0.29 },
  { item: 
     { name: 'Comida',
       description: 'Sabrosos temas de WordPress para tu restaurante o blog gastronómico. Publica menús e instrucciones, comparte recetas y fotos de comidas o publica tus reseñas de restaurantes.' },
    score: 0.52 } ]

```

I didn't want to get into fiddling with thresholds, so I think we should go ahead as-is, but if you wanted to circle back later, the docs are at https://fusejs.io/